docker-app/qfieldcloud/core/tests/test_packages.py

```diff
@@ -655,3 +657,54 @@ def test_collaborator_via_team_can_package(self):
                     "project_qfield_attachments.zip",
                 ],
             )
+
+    def test_outdated_packaged_files_are_deleted(self):
+        cur = self.conn.cursor()
+        cur.execute("CREATE TABLE point (id integer, geometry geometry(point, 2056))")
+        self.conn.commit()
+        cur.execute(
+            "INSERT INTO point(id, geometry) VALUES(1, ST_GeomFromText('POINT(2725505 1121435)', 2056))"
+        )
+        self.conn.commit()
```

Comment:
not too sure how this is needed in this test ? (or is simply expected by `project2.qgs` ?)

Reply:
it is inspected in `project2.qgs`.